src/components/Drawer/internal/DrawerFixture/styled.ts

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import { DrawerTransitionDuration } from "../../constants";
+
+export const Container = styled.div<{ isFixed: boolean }>`
+  cursor: pointer;
+  position: relative;
+  margin-left: ${({ theme }) => theme.spacing * 3}px;
+  font-weight: bold;
+  transform: rotate(${({ isFixed }) => (isFixed ? "none" : "180deg")});
+  transition: transform ${DrawerTransitionDuration}s;
+
+  &::before {
+    content: " ";
```

Comment:
```suggestion
    content: "";
```
空でいいなら`""`だね
width出したいならcssで当てた方が良さそう